backend/src/main/java/com/google/sps/notifiers/Notifier.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main.java.com.google.sps.notifiers;
+
+public class Notifier {
+
+  private String name; /* String representing the user who is to be notified's name */
+
+  public Notifier(String name) {
+    setName(name);
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
```

Comment:
Can calling code set the name once in the constructor call? If so then `this.name` could become a final field.

Reply:
No, it seems like final variables cannot be set but i cant imagine a scenario in which the name will change

Reply:
Then let's take out the setter and make the field final

Reply:
done